salt/modules/win_iis.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft IIS site management via WebAdministration powershell module
+
+:platform:      Windows
+
+'''
+
+
+from __future__ import absolute_import
+
+# Import salt libs
+import salt.utils
+
+# Define the module's virtual name
+__virtualname__ = 'win_iis'
+
+
+def __virtual__():
+    '''
+    Load only on Windows
+    '''
+    if salt.utils.is_windows():
+        return __virtualname__
+    return False
+
+
+def _srvmgr(func):
+    '''
+    Execute a function from the WebAdministration PS module
+    '''
+
+    return __salt__['cmd.run'](
+        'Import-Module WebAdministration; {0}'.format(func),
```

Comment:
It is security focused. The concern is that we do not want somebody to be able to pass in a shell-ism such as `goodcommand && evil_command`. I am not at all familiar with Powershell, so if this is a concern that can be safely mitigated some other way, I'm fine with whatever you come up with.